src/engraving/libmscore/score.cpp

```diff
@@ -535,6 +536,29 @@ void Score::fixTicks()
 
         tick += measureTicks;
     }
+
+    for (const auto& pair : spanner()) {
+        const Spanner* spannerItem = pair.second;
+        if (!spannerItem || !spannerItem->isTempoChangeRanged()) {
+            continue;
+        }
+
+        const TempoChangeRanged* tempoChange = toTempoChangeRanged(spannerItem);
+        if (!tempoChange) {
+            continue;
+        }
+
+        int tickPositionFrom = tempoChange->tick().ticks();
+        BeatsPerSecond currentBps = tempomap()->tempo(tickPositionFrom);
+        BeatsPerSecond newBps = currentBps * tempoChange->tempoChangeFactor();
+
+        std::map<int, double> tempoCurve = TConv::easingValueCurve(tempoChange->ticks().ticks(), 4, std::abs(currentBps.val - newBps.val), tempoChange->easingMethod());
+
+        for (const auto& pair : tempoCurve) {
+            tempomap()->setTempo(tickPositionFrom + pair.first, BeatsPerSecond(currentBps.val + pair.second));
```

Comment:
This sets the tempo markers as type FIX into the map; not sure if we want to use the type::RAMP here instead (which seems to be reserved for supporting this)

Reply:
Although I'm fine with discrete markings here as well; it kind of depends on whether we need to be able to differ between actual discrete fixed markings and such a RAMP marking for export for example

Reply:
I think FIX type is much easier to understand and maintain, so I'd stick with this approach.

However, if we'll want to achieve a lower sampling rate, it would make sense to make the parameter stepCount configurable by user